.semaphore/semaphore.yml

```diff
@@ -0,0 +1,14 @@
+version: v1.0
+name: Protoman
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+blocks:
+  - name: 'Publish gem'
+    run:
+      when: branch = 'master' and change_in(['../lib/global_phone/version.rb'])"
+    task:
+      jobs:
+				- name: Publish new version
```

Comment:
not sure what's goin on here, but these likely need to be aligned as a single tab over underneath the `jobs:` token there.

Reply:
Oh weird. Yeah I'll look into this. It shows up normally on my editor.

Reply:
yeah, not sure what's going on with this but it'll have to be fixed before this ships; I don't think this would be considered valid YAML as-is, meaning semaphore would likely just fail on loading this script